### ISSUE TITLE
[bug] `dom/no-missing-iframe-sandbox` error for `sandbox=""`

### ISSUE THREAD

**Issue by alex-kinokon** [Rel1cx/eslint-react#700]
### Describe the bug

`iframe` can have an empty string `sandbox=""` attribute. This rule is erroneously reporting `Add missing 'sandbox' attribute on 'iframe' component.`.

### Platform and versions

```Text
1.7.1
```
